teachopencadd/talktorials/002_compound_adme/talktorial.ipynb

```diff
@@ -6,11 +6,11 @@
    "source": [
```

Comment:
<p>One could move the first comment here to an extra markdown cell above the code. Same for the cell above.</p><p> </p><p>and maybe rephrase: </p><p>'# Concatenate molecules with Ro5 data'</p><p>-&gt; Concatenate dataframes containing molecule and Ro5 information</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>